pkg/cluster/manager/display.go

```diff
@@ -225,6 +225,124 @@ func (m *Manager) Display(name string, opt operator.Options) error {
 	return nil
 }
 
+// DisplayTiKVLabels display cluster tikv labels
+func (m *Manager) DisplayTiKVLabels(name string, opt operator.Options) error {
```

Comment:
Seems you are referring a pretty old code base, the latest one on master has a JSON output mode available. Please rebase your changes onto master.

Reply:
Master JSON is other feature function，The two do not conflict

Reply:
It is, the label displaying is not compatible with JSON output, so the `display --json` will be broken in some situation.

Reply:
em...，you meaning `display --labels --json` situation , It‘s broken ,  but it output json ，not very good output，because machine ip row is exist null and exist multi row

Reply:
Yes, any new features should not break existing ones unless absolutely necessary.

Reply:
It’s necessary. because yamlmeta file TiKV Labels has poor readability and checkability，I haven’t thought of a better processing method for the time being. The current look and output are friendly to operation and maintenance

Reply:
I mean as `display` now supports JSON output, any change made to it should also support JSON output, otherwise the compatibility is compromised.